modes/id/melamar.md

```
@@ -0,0 +1,114 @@
+# Mode: melamar -- Asisten live untuk formulir lamaran
+
+Mode interaktif untuk saat kandidat mengisi formulir lamaran di Chrome. Membaca apa yang ada di layar, memuat konteks evaluasi lowongan sebelumnya, dan menghasilkan jawaban yang dipersonalisasi untuk setiap pertanyaan formulir.
+
+## Prasyarat
+
+- **Ideal dengan Playwright terlihat**: Dalam mode terlihat, kandidat melihat browser dan Claude bisa berinteraksi dengan halaman.
+- **Tanpa Playwright**: kandidat membagikan tangkapan layar atau menempelkan pertanyaan secara manual.
+
+## Alur kerja
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)